scylla/src/transport/connection.rs

```diff
@@ -428,6 +426,7 @@ impl Connection {
                 } else {
                     // TODO: Handle this error better, for now we drop this
                     // request and return an error to the receiver
+                    warn!("Could not allocate stream id");
```

Comment:
This looks more as an error to me than a warning

Reply:
Thanks, fixed